src/pages/equipo.js

```diff
@@ -1,386 +1,413 @@
-        linkedIn:"https://www.linkedin.com/in/angelo-castro-gamboa/",
-        twitter:"",
-        color:"#ffffff"
-    },
-    {
-        img:"img/team/cristian.jpg",
-        name:"Cristian Castro",
-        position:"Desarrollador C++",
-        bio:"Estamos creando las nuevas tecnologías que harán realmente libres a las personas.",
-        linkedIn:"https://www.linkedin.com/in/cristian-castro-95193a3a/",
-        twitter:"",
-        color:"#f8f8f8"
-    },
-    {
-        img:"img/team/jorge.jpg",
-        name:"Jorge Murillo",
-        position:"Diseñador UX Sr.",
-        bio:"Entender las necesidades es crucial para crear interfases para apps de alta tecnología y mediar entre usuarios, diseñadores y desarrolladores.",
-        linkedIn:"https://www.linkedin.com/in/jmurilloux/",
-        twitter:"https://twitter.com/jmur1ll0",
-        color:"#ffffff"
-    },
-    {
-        img:"img/team/kevin.jpg",
-        name:"Kevin Castillo",
-        position:"Desarrollador Full Stack",
-        bio:"El desarrollo de software me ha dado la oportunidad de aprender y crear nuevas cosas cada día.",
-        linkedIn:"https://www.linkedin.com/in/kevin-castillo-mora/",
-        twitter:"",
-        color:"#f8f8f8"
-    },
-    {
-        img:"img/team/leister.jpg",
-        name:"Leister Álvarez",
-        position:"Desarrollador Jr Backend",
-        bio:"Programar refleja quiénes somos; hagamos algo increíble.",
-        linkedIn:"https://www.linkedin.com/in/leister-francisco-alvarez-campos-65b7801bb/",
-        twitter:"https://twitter.com/leisterac",
-        color:"#ffffff"
-    },
-    {
-        img:"img/team/teto.jpg",
-        name:"Terencio Gómez",
-        position:"Desarrollador Full Stack",
-        bio:"Mantener un equipo feliz puede garantizar un resultado final exitoso.",
-        linkedIn:"https://www.linkedin.com/in/terencio-gomez-14784ba3/",
-        twitter:"",
-        color:"#f8f8f8"
-    }
-]
+  {
+    img: "img/team/adriel.jpg",
+    name: "Adriel Diaz",
+    position: "Desarrollador Full Stack",
+    bio: "Padre orgulloso y desarrollador full-stack.",
+    linkedIn: "https://www.linkedin.com/in/adriel-d%C3%ADaz-08226384/",
+    twitter: "",
+    color: "#f8f8f8",
+  },
+  {
+    img: "img/team/angelo.jpg",
+    name: "Angelo Castro",
+    position: "Desarrollador Jr Full Stack",
+    bio: "Estoy listo para desarrollar ideas innovadoras y enfrentar nuevos desafíos.",
+    linkedIn: "https://www.linkedin.com/in/angelo-castro-gamboa/",
+    twitter: "",
+    color: "#ffffff",
+  },
+  {
+    img: "img/team/jorge.jpg",
+    name: "Jorge Murillo",
+    position: "Diseñador UX Sr.",
+    bio: "Entender las necesidades es crucial para crear interfases para apps de alta tecnología y mediar entre usuarios, diseñadores y desarrolladores.",
+    linkedIn: "https://www.linkedin.com/in/jmurilloux/",
+    twitter: "https://twitter.com/jmur1ll0",
+    color: "#ffffff",
```

Comment:
Please review mobile version, color should be alternated between #f8f8f8 and #ffffff